core/services/vrf/delegate.go

```diff
@@ -117,6 +119,10 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 
 	for _, task := range pl.Tasks {
 		if _, ok := task.(*pipeline.VRFTaskV2); ok {
+			if err := CheckFromAddressMaxGasPrices(jb, chain.Config()); err != nil {
```

Comment:
this is a sanity check at this point, correct? There shouldn't be any way for us to get here..